src/Illuminate/Foundation/Auth/ThrottlesLogins.php

```diff
@@ -23,6 +23,17 @@ protected function hasTooManyLoginAttempts(Request $request)
     }
 
     /**
+     * Determine how many retries left.
+     *
+     * @param Request $request
+     * @return int
+     */
+    protected function retriesLeft(Request $request)
+    {
+        return $this->maxLoginAttempts() - app(RateLimiter::class)->attempts($request->input($this->loginUsername()).$request->ip()) + 1;
```

Comment:
Could you make this line shorted by assigning variables earlier within this method please.